desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js

```diff
@@ -130,26 +164,52 @@ export default ({ introduction, fair_coverage, event, prepare_for_fairs }) =>
           </SectionTitle>
         </Col>
         <Col lg={8} md={8} sm={12} xs={12}>
-          {prepare_for_fairs.articles.map(article =>
+          {prepare_for_fairs.articles.map(article => (
             <Row style={{ marginBottom: 25 }} key={article.title}>
               <Col lg={7} md={7} sm={6} xs={12}>
-                <a href={article.article_url} target='_blank'>
-                  <img style={{ marginBottom: 10, width: '100%' }} src={article.image_url} />
+                <a href={article.article_url} target="_blank">
+                  <img
+                    style={{ marginBottom: 10, width: "100%" }}
+                    src={article.image_url}
+                  />
                 </a>
               </Col>
               <Col lg={5} md={5} sm={6} xs={12}>
-                <a href={article.article_url} style={{ textDecoration: 'none' }} target='_blank'>
-                  <Title titleSize='small' style={{ margin: '0 0 5px', lineHeight: 1 }}>
+                <a
+                  href={article.article_url}
+                  style={{ textDecoration: "none" }}
+                  target="_blank"
+                >
+                  <Title
+                    titleSize="small"
+                    style={{ margin: "0 0 5px", lineHeight: 1 }}
+                  >
                     {article.title}
                   </Title>
-                  <Text textStyle='primary' textSize='small'>
+                  <Text textStyle="primary" textSize="small">
                     {article.author}
                   </Text>
                 </a>
               </Col>
             </Row>
-          )}
+          ))}
         </Col>
       </Row>
+      <Cta>
+        <Container style={{ padding: "20px 0", display: "flex" }}>
```

Comment:
This styling could alternatively live within styled-components in the CTA definition --
```
const CTA = styled.div`
   ${Container} {
      padding: 20px 0;
      display: flex;
   }
`
```